3/Destructuring.js

```diff
@@ -0,0 +1,13 @@
+const sarah = {
```

Comment:
don't use additional variables for the desctructuring. Do it directly with input param